### ISSUE TITLE
JAR files are not present in the Ubuntu / Debian `khiops` desktop packages

### ISSUE THREAD

**Issue by popescu-v** [KhiopsML/khiops#120]
#### Description
When following the official instructions for installing Khiops Desktop on Ubuntu (22.04) - see https://khiops.org/setup/nocode/#__tabbed_1_2 - the JAR files `khiops.jar` and `norm.jar` are not installed in `/usr/share/khiops`, as expected by the `khiops-env` script. Hence, the Java-based Khiops GUI fails to start.

#### Questions/Ideas
- The issue is that in the CMake install script that `cpack` uses there is no `install` directive to include these JAR files in the `KHIOPS` component. Hence, these JAR files are not included in the `khiops_*.*.deb` packages, but are included in the `khiops-unspecified_*.*.deb` packages instead which are not (supposed to be) released.
- The solution is to include these two JAR files in an `install` directive for the `KHIOPS` component.
  - Moreover, we can also exclude the "Unspecified" component from being used for generating the `khiops-unspecified_*.*.deb` package.
- Note: This is expected to also impact RPM packages, in exactly the same way, as the issue seems to be not Debian-related, but CPack-related.
- Note: We need to see how / whether this can be tested on the CI.
### Context
- Khiops version: Khiops 10.2.0
- Log file (use `khiops -e log.txt`): its contents are
   ```
   fatal error : Java Library : Please check that Java version is at least Java 7, that the X server is avai 
   lable under current environment and check access to the norm.jar library in the Java classpath
   ```
- Scenario file (use `khiops -o scenario._kh`): not applicable (calling `khiops` fails to launch the GUI and exits)
- OS description (use `khiops -s`): this command exits with `Unexpected -s flag`.

**Comment by folmos-at-orange** [comment]
We can close this one no ?

**Comment by folmos-at-orange** [comment]
We can close this one @popescu-v , no ?

**Comment by popescu-v** [comment]
IMHO, yes.

**Comment by popescu-v** [comment]
> We can close this one @popescu-v , no ?

IMHO, yes.

**Comment by popescu-v** [comment]
Closed by #121

**Comment by popescu-v** [comment]
Closed by #121
